app/views/api/clidocs/taxa2tree.html.erb

```diff
@@ -7,19 +7,19 @@
       </h1>
     </div>
     <p>
-      The <code>unipept taxa2tree</code> command takes one or more <span class="initialism">NCBI</span> taxon id's as input and returns the taxonomic tree of these taxa as output. All this information is fetched by doing <a href='<%= api_apidocs_taxa2tree_path %>' target='_blank'><span class="initialism">API</span>-requests</a> to the Unipept server.
+      The <code>unipept taxa2tree</code> command takes one or more <span class="initialism">NCBI</span> taxon ID's as input and returns the taxonomic tree of these taxa as output. All this information is fetched by doing <a href='<%= api_apidocs_taxa2tree_path %>' target='_blank'><span class="initialism">API</span>-requests</a> to the Unipept server.
```

Comment:
IDs, not ID's. The `'` is a Dutch thing.

Reply:
Fixed in the next commit :)